### PR TITLE
Add option to ignore prerelease on update

### PR DESCRIPTION
**Pull request by jashandeep-sohi**
Hi,

This adds a `omitPrereleaseDuringUpdate` option, that should allow you to preserve the prerelease state on updates.

### CODE REVIEW

**Comment by ncipollo** [comment]
Could you elaborate a little bit on the use case? Is there a scenario where you would want to create the release with the pre-release flag set to true but omit it when you update the release?

**Comment by jashandeep-sohi** [comment]
To me updating a release is completely orthogonal to its prerelease status. For example, I create a prerelease `v1.2-alpha` in one workflow and in another upload additional assets for that release. That shouldn't automatically mark the release as no longer being a prerelease.

**Comment by ncipollo** [comment]
Ah okay, that makes sense! Thanks for the additional context!

File: README.md [hunk 1]
```diff
@@ -20,6 +20,8 @@ This action will create a GitHub release and optionally upload an artifact to it
 - **omitNameDuringUpdate**: Indicates if the release name should be omitted during updates. The name will still be applied for newly created releases. This will preserve the existing name during updates.
 - **owner**: Optionally specify the owner of the repo where the release should be generated. Defaults to current repo's owner.
 - **prerelease**: Optionally marks this release as prerelease. Set to true to enable.
+- **omitPrereleaseDuringUpdate**: Indicates if the prerelease flag should be omitted during updates. The prerelease flag will still be applied for newly created releases.
```
Comment (ncipollo) [inline README.md:23]: Could we move this up with the rest of the `omit**` inputs? This list is roughly alphabetical.

File: action.yml [hunk 2]
```diff
@@ -71,6 +71,10 @@ inputs:
     description: "Optionally marks this release as prerelease. Set to true to enable."
     required: false
     default: ''
+  omitPrereleaseDuringUpdate:
```
Comment (ncipollo) [inline action.yml:74]: Same here, could we move this up with the reset of the `omit*` inputs.

File: src/Inputs.ts [hunk 3]
```diff
@@ -129,6 +130,14 @@ export class CoreInputs implements Inputs {
...
     }
 
+    private static get omitPrereleaseDuringUpdate(): boolean {
+        return core.getInput('omitPrereleaseDuringUpdate') == 'true'
+    }
+
+    get updatedPrerelease(): boolean | undefined {
+        if (CoreInputs.omitPrereleaseDuringUpdate) return undefined
+        return this.prerelease
+    }
```
Comment (ncipollo) [inline src/Inputs.ts:140]: Could we also update the `prerelease` getter to `createdPrerelease` so it's consistent with the other inputs that have `created` vs `updated` variations?

**Review by ncipollo — CHANGES_REQUESTED** [review]
The new functionality look great! I had a few small nits but after those are addressed I think this will be good to merge!

**Review by ncipollo — APPROVED** [review]
Looks great, thanks for making those changes!

**Pull request merged**